### PR TITLE
Fix benchmark configs

### PR DESCRIPTION
**Pull request by ernestum**
## Description

- [x] Fixes issue with the benchmark config files
- [x] Improves on the readability of the benchmark tests
- [x] Adds missing `imitation.algorithms.dagger.ExponentialBetaSchedule` (@Rocamonde could you do this?)

More details in #664.

Fixes issue #664 which was introduced in #653. 

## Testing

Improved test for the benchmarks.

### CODE REVIEW

**Comment by Rocamonde** [comment]
(y) Will review tomorrow.

**Review by Rocamonde — APPROVED** [review]
LGTM.

**Comment by ernestum** [comment]
Thanks for the review @Rocamonde !
I still need the missing `imitation.algorithms.dagger.ExponentialBetaSchedule` file from you (see description of the PR). Otherwise the test can not pass.

**Comment by Rocamonde** [comment]
Unfortunately I don't really have this file! You should ping @taufeeque9 who ran the experiments & gave me the JSON's.

**Comment by ernestum** [comment]
All right then please @taufeeque9 please add the files to this PR when you get around to it.

**Comment by AdamGleave** [comment]
@Rocamonde I think this is ready for another review now that Taufeeque has added the missing beta scheduler

**Review by Rocamonde — APPROVED** [review]
LGTM, just some minor comments. please do take a look before merging

File: tests/test_benchmarking.py [hunk 1]
```diff
@@ -10,24 +8,39 @@
...
+ENVIRONMENTS = [
+    "seals_walker",
+    "seals_ant",
+    "seals_half_cheetah",
+    "seals_hopper",
+    "seals_swimmer",
+]
 
-@pytest.mark.parametrize(
-    "command_name",
```
[the remark this answers is not in the document]
Comment (ernestum) [reply tests/test_benchmarking.py:15]: I think I don't get your question. What are you referring to? The [parameterized](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#parametrize-basics) "command_name"?

File: tests/test_benchmarking.py [hunk 2]
```diff
@@ -10,24 +8,39 @@
...
+    if algorithm in ("bc", "dagger"):
+        experiment = train_imitation.train_imitation_ex
+    elif algorithm in ("airl", "gail"):
+        experiment = train_adversarial.train_adversarial_ex
+    else:
+        raise ValueError(f"Unknown algorithm: {algorithm}")  # pragma: no cover
+
+    config_name = f"{algorithm}_{environment}"
+    config_file = str(
+        BENCHMARKING_DIR / f"example_{algorithm}_{environment}_best_hp_eval.json",
```
[the remark this answers is not in the document]
Comment (ernestum) [reply tests/test_benchmarking.py:38]: Yes. But that is 20 of 21 configs right now.

**Pull request merged**